sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MainActivity.java

```diff
@@ -273,6 +274,11 @@ public void run() {
           CoroutinesUtil.INSTANCE.throwInCoroutine();
         });
 
+    binding.enableReplayDebugMode.setOnClickListener(
+        view -> {
+          ReplayIntegration.enableDebugMasking();
```

Comment:
Wondering if we should expose it via `Sentry.replay.enable/disableDebugMaskingOverlay()`? We could introduce `IReplayApi` similar to `ILoggerApi` with this single method. And then we already have [ReplayController](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/ReplayController.java) which is accessible via `SentryOptions` and we could also access it this way from within `ScreenshotRecorder` e.g. `options.replayController.isDebugMaskingEnabled`

Reply:
Yes, let's do that!